crt_portal/cts_forms/signals.py

```diff
@@ -42,10 +42,15 @@ def format_data_message(action, current_request, instance):
 
 
 @receiver(post_save, sender=User)
-def save_user(sender, instance, **kwargs):
-    current_request = CrequestMiddleware.get_request()
-    message = format_user_message('saved', current_request, instance)
-    logger.info(message)
+def save_user(sender, instance, created, **kwargs):
+    if created:
+        current_request = CrequestMiddleware.get_request()
+        message = format_user_message('created', current_request, instance)
+        logger.info(message)
+    else:
+        current_request = CrequestMiddleware.get_request()
+        message = format_user_message('saved', current_request, instance)
+        logger.info(message)
```

Comment:
Great addition to differentiate this log output! Opportunity here to use a ternary and cut a bit of the duplication with something like:

```python
def save_user(sender, instance, created, **kwargs):
    action = 'created' if created else 'saved'
    current_request = CrequestMiddleware.get_request()
    message = format_user_message(action, current_request, instance)
    logger.info(message)
```